cfe_internal/enterprise/federation/federation.cf

```diff
@@ -24,19 +24,31 @@ bundle agent config
         int => "20",
         comment => "Dump data on the feeders every 20 minutes";
 
+      # TODO: don't hard-code cftransport user
+      "transport_user" -> { "ENT-4610" } string => "cftransport";
+      "transport_home"
+        string => "$(cfengine_enterprise_federation:config.federation_dir)/cftransport";
+
+    config_present::
+      "data" data => readjson( $(path) );
+
   classes:
     enterprise_edition.(policy_server|am_policy_hub)::
-      "enabled"
-        expression => fileexists( $(path) ),
+      "config_present"
+        expression => fileexists( $(path) );
+    config_present::
+      "enabled" expression => or(strcmp("on", "$(data[target_state])"),
+                                 strcmp("paused", "$(data[target_state])")),
+        scope => "namespace";
+      "am_off" expression => strcmp("off", "$(data[target_state])"),
+        scope => "namespace";
+      "am_on" expression => strcmp("on", "$(data[target_state])"),
         scope => "namespace";
```

Comment:
Just pondering
```
vars:
"state" string => "$(data[target_state])";
classes:
"am_$(state)";
```

Reply:
Yeah, that makes some sense for sure. Definitely lets the set of values grow.